build.sbt

```diff
@@ -44,6 +44,11 @@ lazy val akkaHttpVersion = "10.2.10"
 lazy val circeVersion = "0.14.3"
 lazy val testContainersVersion = "0.40.10"
 
+dependencyOverrides ++= Seq(
```

Comment:
This is in the `build.sbt` file twice. Also on line 89: https://github.com/nationalarchives/tdr-consignment-api/blob/master/build.sbt#L89. Probably should remove one of them

Reply:
I'm blind, I didn't notice it was already in there. I've removed the one I added.